README.md

```diff
@@ -1,10 +1,14 @@
 # Publish to GitHub wiki action
 
-[![Test](https://github.com/spenserblack/actions-wiki/actions/workflows/test.yml/badge.svg)](https://github.com/spenserblack/actions-wiki/actions/workflows/test.yml)
+[![Test](https://github.com/jcbhmr/actions-wiki/actions/workflows/test.yml/badge.svg)](https://github.com/jcbhmr/actions-wiki/actions/workflows/test.yml)
```

Comment:
Whoops, this should probably be changed back :laughing:

Reply:
Ooops 😜